.github/scripts/trymerge.py

```diff
@@ -1291,6 +1291,21 @@ def merge(pr_num: int, repo: GitRepo,
 
     check_for_sev(org, project, force)
 
+    gh_post_pr_comment(org, project, pr.pr_num, explainer.get_merge_message(land_check_commit))
+    if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days:
+        raise RuntimeError("This PR is too stale; the last push date was more than 3 days ago. Please rebase and try again.")
+
+    try:
+        # Important: check for merge rule once before starting land checks
+        # because we want to make sure that only approved PRs can start CI
+        # jobs. If there's missing approval, a RuntimeError will be raised
+        # here to stop the merge process right away
+        find_matching_merge_rule(pr, repo)
+    except MandatoryChecksMissingError:
+        # Missing mandatory checks is fine here because the merge job hasn't
+        # been started yet
+        pass
```

Comment:
Occams Razor: do not throw, but simply pass "force" flag to validate only reviews/CLA part of the rule
```suggestion
    # Important: check for merge rule once before starting land checks
    # because we want to make sure that only approved PRs can start CI
    # jobs. If there's missing approval, a RuntimeError will be raised
    # here to stop the merge process right away
    find_matching_merge_rule(pr, repo, force=True)
```

Reply:
Ohhh, this is wicked. I didn't realize that the `force` flag behaves that way. But you're right, setting `force` to true will only include CLA check in the list of required checks and achieves what we need here.